### PR TITLE
Filter out bounding boxes with width or height below parameters

### PR DESCRIPTION
**Pull request by jangop**
While this does not fix the problems highlighted in #1322, it does help with filtering out further problematic bounding boxes.

### CODE REVIEW

File: albumentations/core/bbox_utils.py [hunk 1]
```diff
@@ -441,20 +459,32 @@ def filter_bboxes(
         min_area: Minimum area of a bounding box. All bounding boxes whose visible area in pixels.
             is less than this value will be removed. Default: 0.0.
         min_visibility: Minimum fraction of area for a bounding box to remain this box in list. Default: 0.0.
+        min_width: Minimum width of a bounding bpx. All bounding boxes whose width is
```
Comment (Dipet) [inline albumentations/core/bbox_utils.py:462]: Typo: `bpx` -> `box`

File: albumentations/core/bbox_utils.py [hunk 2]
```diff
@@ -62,11 +62,15 @@ def __init__(
...
         min_visibility: float = 0.0,
+        min_width: float = 0.0,
+        min_height: float = 0.0,
         check_each_transform: bool = True,
     ):
         super(BboxParams, self).__init__(format, label_fields)
         self.min_area = min_area
         self.min_visibility = min_visibility
+        self.min_width = min_width
+        self.min_height = min_height
```
Comment (Dipet) [inline albumentations/core/bbox_utils.py:73]: Please, add new args to `_to_dict`

**Review by Dipet — CHANGES_REQUESTED** [review]

File: albumentations/core/bbox_utils.py (hunk 1 above)
Comment (jangop) [reply albumentations/core/bbox_utils.py]: [702b31e](https://github.com/albumentations-team/albumentations/pull/1327/commits/702b31ec7fca39a8cbc9cb532e6cc334265c885a)

File: albumentations/core/bbox_utils.py (hunk 2 above)
Comment (jangop) [reply albumentations/core/bbox_utils.py:73]: I had not noticed `_to_dict`. Thanks.

da83ed0

File: albumentations/core/bbox_utils.py [hunk 3]
```diff
@@ -62,11 +62,15 @@ def __init__(
         label_fields: Optional[Sequence[str]] = None,
         min_area: float = 0.0,
         min_visibility: float = 0.0,
+        min_width: float = 0.0,
+        min_height: float = 0.0,
```
Comment (Dipet) [inline albumentations/core/bbox_utils.py:66]: Please, copy description of this args from `filter_bboxes` to this docstring too.

Comment (jangop) [reply albumentations/core/bbox_utils.py:70]: c850f22

**Review by Dipet — APPROVED** [review]

**Pull request merged**